azerothcore/mod-ah-bot
# Change of the selling priority, following item quality instead of a random pick

## Changes Proposed:
- The selection of items to sold in the auction house has been changes from a random pick to a priority approach. Instead of relying on a random pick which can waste cycles, the configured quotas are met in order of quality, starting from the poor items and ending with artifacts. This cause the bot to pick items in order.

## Issues Addressed:

- No fixes addressed

## SOURCE:


## Tests Performed:

- The code build with no errors.
- Tested on my private server; there are no really visible changes since the configured quotas keeps the number of items at the same level.


## How to Test the Changes:


1. Pull the code and compile it within your installation.
2. Configure the bot to sell low items per cycle.
3. Observe the auction house growing; you will see uncommon items spawning firsts, as it should be in a normal market, and more uncommon and rare items appears later during the day.
